src/pf.cpp

```diff
@@ -33,7 +36,14 @@ std::string get_input_line() {
 
 fs::path base_dir_env() {
     auto ptr = std::getenv("PF_BASE_DIR");
-    return ptr ? ptr : fs::current_path();
+    if (ptr) {
+        return ptr;
+    }
+
+    // Attempt to detect the base directory
+    return pf::detect_base_dir()
+        // Just use the cwd if cannot detect base dir
+        .value_or(fs::current_path());
```

Comment:
Should this logic be part of `base_dir_env()`? I think it probably should not; this should probably be part of the `get_base_dir()` logic

Reply:
Yeah, or rename this to something other than `base_dir_env()`